packages/core/src/routes/setting.ts

```diff
@@ -0,0 +1,32 @@
+import { Settings } from '@logto/schemas';
+
+import RequestError from '@/errors/RequestError';
+import koaGuard from '@/middleware/koa-guard';
+import { getSetting, updateSettingById } from '@/queries/setting';
+
+import { AuthedRouter } from './types';
+
+export default function settingRoutes<T extends AuthedRouter>(router: T) {
+  router.get('/settings', async (ctx, next) => {
+    try {
+      ctx.body = await getSetting();
+    } catch {
+      throw new RequestError('entity.not_exists');
+    }
+
+    return next();
+  });
+
+  router.patch(
+    '/settings',
+    koaGuard({
+      body: Settings.guard,
+    }),
+    async (ctx, next) => {
+      const { body: setting } = ctx.guard;
+      ctx.body = await updateSettingById(setting.id, setting);
```

Comment:
A partical update will match the http method "Patch".

Reply:
you mean partial update?